schemas/v1/4_surveyed_peer_agents.go

```diff
@@ -0,0 +1,23 @@
+package v1
+
+// Schema patch 4 adds surveyed peer agents schema
+
+func init() {
+	patches.Register(
+		4,
+		`
+	-- ----------------------------------------------------------------
+	-- Name: surveyed_peer_agents
+	-- Model: surveyed.PeerAgent
+	-- Growth: N/A
+	-- ----------------------------------------------------------------
+
+	CREATE TABLE {{ .SchemaName | default "public"}}.surveyed_peer_agents (
+		observed_at       timestamp with time zone NOT NULL,
+		raw_agent         text NOT NULL,
+		normalized_agent  text NOT NULL,
+    	count             bigint NOT NULL
+	);
+	ALTER TABLE ONLY {{ .SchemaName | default "public"}}.surveyed_peer_agents ADD CONSTRAINT surveyed_peer_agents_pkey PRIMARY KEY (observed_at, raw_agent);
```

Comment:
Should we add comments to the schema?

Reply:
You might also consider including comments for `gapfill_reports` and `chain_consensus`. 🙏